src/pages/blogs/how_shorebird_works.md

```diff
@@ -0,0 +1,219 @@
+# How Shorebird Works
+
+## Code Push
+
+Code push, sometimes called “over the air updates”, is a way of updating
+application code in production so that all your users are always running the
+latest code – just like how your web application works. Code push for Flutter is
+one of the top 50 most upvoted issues across all of GitHub. Code push solutions
+have long been used by mobile developers to make app development more like web
+development. Existing solutions have typically relied on WebViews or Lua
+scripts, and require developers to use different languages and frameworks for
+different parts of their applications. Such solutions also implicitly require
+developers to be able to predict where they will have bugs in their code, since
+only some parts of their applications are updatable and others not. At Shorebird
+when we sat down to build Code Push for Flutter, we wanted to build something
+better.
+
+Shorebird’s code push for Flutter allows developers to update their Flutter apps
+instantly, over the air, deploying fixes directly to end users’ devices. Our
+solution takes < 5 minutes to integrate and requires no code changes.
+Shorebird’s code push can update any Dart code in your app. We’ve designed our
```

Comment:
```suggestion

Shorebird’s code push can update any Dart code in your app. We’ve designed our
```

Reply:
Maybe we could also start this paragraph differently? It starts with the exact same words than the one before.

Reply:
I'm not seeing the duplication?